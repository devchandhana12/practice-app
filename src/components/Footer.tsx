import React from "react";
const Footer: React.FC<any> = () => {
  return <div>im a footer to your site</div>;
};
export default Footer;
