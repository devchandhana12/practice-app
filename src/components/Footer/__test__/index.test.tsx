import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";
import Footer from "..";

afterEach(cleanup);
it("should match the snapshot of footer", () => {
  const { asFragment, getByText } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
  // eslint-disable-next-line testing-library/prefer-screen-queries
  const content = getByText("im a footer to your site");
  expect(content).toBeInTheDocument();
});
