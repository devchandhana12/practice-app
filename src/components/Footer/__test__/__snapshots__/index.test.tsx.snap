// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot of footer 1`] = `
<DocumentFragment>
  <div>
    im a footer to your site
  </div>
</DocumentFragment>
`;
